### DISCUSSION TITLE
any way to trigger spotlight from spacehammer?

### DISCUSSION THREAD

**Discussion by jf** [agzam/spacehammer#198] [Q&A] [answered]
let's say that I would prefer (given the placement of the keys) to use `Cmd-space` instead to trigger spacehammer. And that instead of using Alfred (which I do not use), I want to have `Cmd-space`, then `space` again trigger Spotlight instead of Alfred. I've tried doing `(activator "Spotlight")` instead of `(activator "Alfred 4")`, but that doesn't work. How do I use spacehammer to trigger Spotlight?

**Comment by Grazfather** [comment, marked as answer]
Assuming you already changed the binding for Spotlight from the default, try sending that set of keys using `(hs.eventtap.KeyStoke ...)`, which you can bind with `#(hs.eventtap.KeyStroke ...)` e.g. `#(hs.eventtap.KeyStroke [:cmd] :space)`, but first I'd see if it works in the console.

**Reply by jf** [reply]
thank you! that was helpful and I managed to do this.
